articles/azure-maps/migrate-from-bing-maps-web-services.md

```diff
@@ -679,3 +677,83 @@ Learn more about the Azure Maps REST services.
 
 [Azure Maps account]: quick-demo-map-app.md#create-an-azure-maps-account
 [subscription key]: quick-demo-map-app.md#get-the-subscription-key-for-your-account
+
+[Search]: /rest/api/maps/search
+[Route directions]: /rest/api/maps/route/getroutedirections
+[Route Matrix]: /rest/api/maps/route/postroutematrixpreview
+[Render]: /rest/api/maps/render/getmapimage
+[Route Range]: /rest/api/maps/route/getrouterange
+[POST Route directions]: /rest/api/maps/route/postroutedirections
+[Route]: /rest/api/maps/route
+[Time Zone]: /rest/api/maps/timezone
+[Elevation]: /rest/api/maps/elevation
+
+[Azure Maps Creator]: creator-indoor-maps.md
+[Spatial operations]: /rest/api/maps/spatial
+[Map Tiles]: /rest/api/maps/render/getmaptile
+[Map imagery tile]: /rest/api/maps/render/getmapimagerytile
+[Batch routing]: /rest/api/maps/route/postroutedirectionsbatchpreview
+[Traffic]: /rest/api/maps/traffic
+[Geolocation API]: /rest/api/maps/geolocation/get-ip-to-location
+[Weather services]: /rest/api/maps/weather
+
+[Best practices for Azure Maps Search service]: how-to-use-best-practices-for-search.md
+[Best practices for Azure Maps Route service]: how-to-use-best-practices-for-routing.md
+
+[free account]: https://azure.microsoft.com/free/?azure-portal=true
+[manage authentication in Azure Maps]: how-to-manage-authentication.md
+
+[Free-form address geocoding]: /rest/api/maps/search/getsearchaddress
+[Structured address geocoding]: /rest/api/maps/search/getsearchaddressstructured
+[Batch address geocoding]: /rest/api/maps/search/postsearchaddressbatchpreview
+[Fuzzy search]: /rest/api/maps/search/getsearchfuzzy
+[Fuzzy batch search]: /rest/api/maps/search/postsearchfuzzybatchpreview
+
+[Authentication with Azure Maps]: azure-maps-authentication.md
+[Localization support in Azure Maps]: supported-languages.md
+[Azure Maps supported views]: supported-languages.md#azure-maps-supported-views
+
+[Address reverse geocoder]: /rest/api/maps/search/getsearchaddressreverse
+[Cross street reverse geocoder]: /rest/api/maps/search/getsearchaddressreversecrossstreet
+[Batch address reverse geocoder]: /rest/api/maps/search/postsearchaddressreversebatchpreview
+
+[POI search]: /rest/api/maps/search/getsearchpoi
+[POI category search] /rest/api/maps/search/getsearchpoicategory
+[Calculate route]: /rest/api/maps/route/getroutedirections
```

Comment:
Lines 720-722 show up in the viewable article. Looks like you're missing a colon in line 721.